spec/models/manageiq/providers/azure/cloud_manager/targeted_refresher_spec.rb

```diff
@@ -10,7 +10,7 @@
         context "with additional settings #{additional_settings}" do
           before(:each) do
             @refresh_settings = refresh_settings.merge(:allow_targeted_refresh => true)
-            @refresh_settings.merge(additional_settings)
```

Comment:
@bronaghs @djberg96 this was the issue, the settings were not merged, so we were not testing the treshold that switches to collect all API records and filter in ruby vs. collect records by 1 from the API.